src/main/java/teammates/ui/controller/InstructorSearchPageData.java

```diff
@@ -247,18 +250,55 @@ private void setSearchStudentsTables(StudentSearchResultBundle studentSearchResu
     }
     
     /*************** Create data structures for student search results ********************/
-    private List<StudentRow> createStudentRows(String courseId, 
-                                               StudentSearchResultBundle studentSearchResultBundle) {
-        List<StudentRow> rows = new ArrayList<StudentRow>();      
+    private List<StudentListSectionData> createStudentRows(String courseId, 
+                                                           StudentSearchResultBundle studentSearchResultBundle) {
+        List<StudentListSectionData> rows = new ArrayList<StudentListSectionData>();      
         List<StudentAttributes> studentsInCourse = filterStudentsByCourse(
                                                        courseId, studentSearchResultBundle);
-        
+        Map<String, List<String>> sectionNameToTeamNameMap = new HashMap<String, List<String>>();
+        Map<String, List<StudentAttributes>> teamNameToStudentsMap = new HashMap<String, List<StudentAttributes>>();
+        Map<String, String> emailPhotoUrlMapping = new HashMap<String, String>();
```

Comment:
To keep the naming consistent, perhaps you could name it `emailToPhotoUrlMap` instead